src/ProgressBar.js

```diff
@@ -164,7 +160,5 @@ class ProgressBar extends React.Component {
 ProgressBar.propTypes = propTypes;
 ProgressBar.defaultProps = defaultProps;
 
-export default setBsClass(
-  'progress-bar',
-  bsStyles(Object.values(State), ProgressBar)
-);
+export default createBootstrapComponent (ProgressBar, 'progress-bar');
```

Comment:
in order to check the right thing above lets first put this in a local identifier before exporting like: 
```
const DecoratedProgressBar = createBootstrapComponent (ProgressBar, 'progress-bar');
export default DecoratedProgressBar
```


Then at [line](https://github.com/react-bootstrap/react-bootstrap/pull/3065/files#diff-bf9af2152e797180921eba199402e1c3L39) we can check `<DecoratedProgressNav>` and the tests should work again, you might need to make a local exception for eslint if it complains about use-before-define like: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments